ui/apps/platform/cypress.config.js

```diff
@@ -7,7 +7,8 @@
  */
 
 module.exports = {
-    blockHosts: ['*.*'], // Browser options
+    // TODO: can we allow the ocp console but block other internet access? do we need to?
+    // blockHosts: ['*.*'], // Browser options
```

Comment:
I did find this https://issues.redhat.com/browse/ROX-1865 and this https://github.com/stackrox/rox/pull/2304 from when this option was first added.

My initial thought is that it is probably fine to remove this? It only blocks external requests in CI, so I don't see why it would be a problem if the UI doesn't rely on external requests.

@alwayshooin do you have any context on this somewhat ancient change or opinions on whether or not we can remove this?